modules/cluster-logging-forwarding-json-logs-to-the-default-elasticsearch.adoc

```diff
@@ -28,15 +28,15 @@ pipelines:
   parse: json
 ----
 
-. Optional: Use `structuredTypeKey` to specify one of the log record fields, as described in the preceding topic, xref:../logging/cluster-logging-enabling-json-logging.adoc#cluster-logging-configuration-of-json-log-data-for-default-elasticsearch_cluster-logging-enabling-json-logging[Configuring JSON log data for Elasticsearch]. Otherwise, remove this line.
+. Optional: Use `structuredTypeKey` to specify one of the log record fields, as described in the documentation about *Configuring JSON log data for Elasticsearch*. Otherwise, remove this line.
 
-. Optional: Use `structuredTypeName` to specify a `<name>`, as described in the preceding topic, xref:../logging/cluster-logging-enabling-json-logging.adoc#cluster-logging-configuration-of-json-log-data-for-default-elasticsearch_cluster-logging-enabling-json-logging[Configuring JSON log data for Elasticsearch]. Otherwise, remove this line.
+. Optional: Use `structuredTypeName` to specify a `<name>`, as described in the documentation about *Configuring JSON log data for Elasticsearch*. Otherwise, remove this line.
```

Comment:
I don't think this should be bold. Two suggestions:

```suggestion
. Optional: Use `structuredTypeName` to specify a `<name>`, as described in "Configuring JSON log data for Elasticsearch". Otherwise, remove this line.
```

```suggestion
. Optional: Use `structuredTypeName` to specify a `<name>`, as described in the documentation about configuring JSON log data for Elasticsearch. Otherwise, remove this line.
```